examples/planar.js

```diff
@@ -0,0 +1,76 @@
+// # Planar (EPSG:3946) viewer
```

Comment:
in browser `/examples/planar.html` have error 
`Uncaught ReferenceError: bbox is not defined`